website_event_attendee_signup/static/description/index.html

```diff
@@ -38,54 +38,87 @@ <h5 class="oe_slogan" style="font-size:16px;">
 					  <p class="oe_mt32">
 						<font size="3px">As a result, each participant will receive an email with a link to register on the portal.<font>
 					  </font></font></p>
+					  <p class="oe_mt32">
+						<font size="3px">If there several attendees in one checkout, then every person will receive the email to continue signup process.<font>
+					  </font></font></p>
 					</div>
 					<div class="oe_row_img oe_centered">
 					  <img class="oe_demo oe_picture oe_screenshot" src="events-custom-fields-10.png">
 					</div>
 				   </div>
 				</section>
 
-				
 				<section class="oe_container oe_dark">
 				  <div class="oe_row oe_spaced">
-					<h2 class="oe_slogan" style="color:#875A7B;">Free Support</h2>
-					<h3 class="oe_slogan">You will get free support and assistance in case of any issues</h3>
-				  </div>
-				</section>
-
-				<section class="oe_container">
-					<div class="oe_row oe_spaced">
-						<div class="oe_span8">
-							<h2>Need our service?</h2>
-							<p class="oe_mt32">Contact us by <a href="mailto:it@it-projects.info">email</a> or fill out <a href="https://www.it-projects.info/page/website.contactus" target="_blank">request form</a></p>
-							<ul>
-								<li><a href="mailto:it@it-projects.info">it@it-projects.info <i class="fa fa-envelope-o"></i></a></li>
-								<li><a href="https://www.it-projects.info/page/website.contactus" target="_blank">https://www.it-projects.info/page/website.contactus <i class="fa fa-list-alt"></i></a></li>
-							</ul>
-						</div>
-						<div class="oe_span4">
-							<div class="stamp" style="width:200px;">
-								<div style="margin-top: 15px;
-											position: relative;
-											font-family:'Vollkorn', serif;
-											font-size: 16px;
-											line-height: 25px;
-											text-transform: uppercase;
-											font-weight: bold;
-											color: #75526b;
-											border: 3px dashed #75526b;
-											float: left;
-											padding: 4px 12px;
-											-webkit-transform: rotate(-6deg);
-											-o-transform: rotate(-6deg);
-											-moz-transform: rotate(-6deg);
-											-ms-transform: rotate(-6deg);">
-									Tested on Odoo<br>10.0
-								</div>
-							</div>
-						</div>
+					  <p class="oe_mt32">
+						<font size="3px">If there several attendees in one checkout, then every person will receive the email to continue signup process.<font>
+					  </font></font></p>
 					</div>
-				</section>
+					<div class="oe_row_img oe_centered">
+					  <img class="oe_demo oe_picture oe_screenshot" src="events-custom-fields-101.png">
```

Comment:
events-custom-fields-101.png -- currently contains template variables. Use Preview button to see rendered email